pkg/karmadactl/get.go

```diff
@@ -51,6 +51,22 @@ var (
 	noPushModeMessage = "The karmadactl get command now only supports Push mode, [ %s ] are not push mode\n"
 	getShort          = `Display one or many resources`
 	defaultKubeConfig = filepath.Join(os.Getenv("HOME"), ".kube/config")
+
+	getExample = `
+	# List all pods in ps output format
+	karmadactl get pods
```

Comment:
We can't seem to use `karmadactl` directly. Because we also have `kubectl karmada`.

Reply:
Yeah, that's a thing.

```
# ./kubectl-karmada get --help 
Display one or many resources

Usage:
  karmada get [NAME | -l label | -n namespace]  [flags]

Examples:

	# List all pods in ps output format
	karmadactl get pods

	# List a single replicasets controller with specified NAME in ps output format
	karmadactl get replicasets nginx

	# List deployments in JSON output format, in the "v1" version of the "apps" API group
	karmadactl get deployments.v1.apps

	# List all replication controllers and services together in ps output format
	karmadactl get rs,services

	# List one or more resources by their type and names
	karmadactl get rs/nginx-cb87b6d88 service/kubernetes

```